targets/targets.json

```diff
@@ -3520,5 +3520,16 @@
         "device_name": "TMPM066FWUG",
         "detect_code": ["7011"],
         "release_versions": ["5"]
+    },
+    "WUNDERBAR": {
+        "inherits": ["MCU_K24F1M"],
+        "release_versions": ["5"],
+        "macros_add": ["TARGET_WUNDERBAR"],
```

Comment:
This should not be needed, target should prodice this macro

Reply:
Ok, I'll test without this macro and change accordingly.

Reply:
Works perfectly. Done!